### ISSUE TITLE
Public overview of CfP entries

### ISSUE THREAD

**Issue by naltatis** [pretalx/pretalx#1557]
### Problem and impact

We are using pretalx for an internal conference scenario. The CfP form is a great way to collect all talks in a structured ways. However, we want to have a publicly accessible view of all CfP entries so far. That ways potential speakers can see if a topic is already covered, join forces with others or get new ideas for topics that might be interesting. We don't to a formal review process. Everything gets accepted.

We could create a review account for all employees, but this would be a lot of work and hassle. We could also "accept/confirm" all new entries and assign them to a fake schedule that we then publish, but this is also quite clumsy. 

### Proposed solution / feature

Provide an new option to share a list list-view of all CfP entries with their public information.

**Comment by rixx** [comment]
I don't think this will be a feature that will appear in pretalx at its core, at least for the foreseeable future, as it's very far outside of the typical conference requirements. (Organisers of other events interested in this: feel free to comment to prove me wrong, of course!)

Options you have with pretalx right now:

- You could use the "[public-voting](https://github.com/pretalx/pretalx-public-voting/)" plugin, which would still require people to input their email address (though only once). This is of course a bit focused on the voting aspect, but it does provide a list of submitted proposals.
- You could mark all incoming proposals as "featured", which would publish them immediately on the featured preview page. While this still requires an effort on your end (marking talks as they get submitted), it's the intended way to have a preview of proposals without accepting (all of) them.

Hope this helps – and sorry for closing the issue, at least for now.

**Comment by naltatis** [comment]
Thanks so much @rixx. The "featured" way is perfect. Checking the "featured" checkbox is not much work and the entries appear in a nice list linked from the proposal page. That works for us!
